org/pr/milestone.ts

```diff
@@ -1,19 +1,32 @@
 import {warn, danger} from "danger";
 
 export default async () => {
-    // Skip for release PRs
     const githubLabels = danger.github.issue.labels;
+    const currentPR = await danger.github.api.pulls.get(danger.github.thisPR);
```

Comment:
Does this work? 
Looking at the DSL, it seems that there's no `get` method that accepts only the PR number as parameter. It may still work if the other parameters are provided by the context/wrapper, so I just want to double check that it has been verified.

Reply:
@loremattei Unfortunately I don't know if this works or not. However, `danger.github.thisPR` has the following comment in [GitHubDSL's documentation](https://danger.systems/js/reference.html#GitHubDSL):

```
GitHubDSL {
  ...
  // The PR metadata specifically formatted for using with the GitHub API client
  thisPR: GitHubAPIPR
  ...
}
```

And here is the [documentation for GitHubAPIPR](https://danger.systems/js/reference.html#GitHubAPIPR) which it links to:

```
// Provides the current PR in an easily used way for params in `github.api` calls
GitHubAPIPR {
  // The PR number
  number: number
  // The repo owner
  owner: string
  // The repo name
  repo: string
}
```

Am I maybe missing something?

Reply:
Ah, cool. This is perfect and matches the function signature: https://octokit.github.io/rest.js/v17#pulls-get. 👍